.github/labeler.yml

```diff
@@ -1,140 +1,183 @@
-  - '**/*fstab-generator*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*fstab-generator*'
 growfs:
-  - '**/*growfs*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*growfs*'
 hwdb:
-  - hwdb.d/**/*
+  - changed-files:
+    - any-glob-to-any-file: 'hwdb.d/**/*'
 journal:
-  - src/journal/*
-  - src/libsystemd/sd-journal/*
+  - changed-files:
+    - any-glob-to-any-file: ['src/journal/*', 'src/libsystemd/sd-journal/*']
 journal-remote:
-  - src/journal-remote/*
+  - changed-files:
+    - any-glob-to-any-file: 'src/journal-remote/*'
 meson:
-  - meson_options.txt
-  - '**/meson.build'
+  - changed-files:
+    - any-glob-to-any-file: ['meson_options.txt', '**/meson.build']
 mkosi:
-  - .mkosi/*
-  - mkosi.build
+  - changed-files:
+    - any-glob-to-any-file: ['.mkosi/*', 'mkosi.build']
 network:
-  - src/libsystemd-network/**/*
-  - src/network/**/*
+  - changed-files:
+    - any-glob-to-any-file: ['src/libsystemd-network/**/*', 'src/network/**/*']
 portable:
-  - src/portable/**/*
+  - changed-files:
+    - any-glob-to-any-file: 'src/portable/**/*'
 rc-local-generator:
-  - src/rc-local-generator/*
+  - changed-files:
+    - any-glob-to-any-file: 'src/rc-local-generator/*'
 repart:
-  - '**/*repart*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*repart*'
 resolve:
-  - '**/*resolve*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*resolve*'
 rfkill:
-  - '**/*rfkill*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*rfkill*'
 rpm:
-  - src/rpm/*
+  - changed-files:
+    - any-glob-to-any-file: 'src/rpm/*'
 run:
-  - src/run/*
-  - man/systemd-run*
+  - changed-files:
+    - any-glob-to-any-file: ['src/run/*', 'man/systemd-run*']
 sd-boot/sd-stub/bootctl:
-  - src/boot/**/*
-  - man/bootctl*
-  - man/systemd-boot.xml
+  - changed-files:
+    - any-glob-to-any-file: ['src/boot/**/*', 'man/bootctl*', 'man/systemd-boot.xml']
 sd-bus:
-  - '**/sd-bus*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-bus*/**'
 sd-daemon:
-  - '**/sd-daemon*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-daemon*/**'
 sd-device:
-  - '**/sd-device*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-device*/**'
 sd-event:
-  - '**/sd-event*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-event*/**'
 sd-hwdb:
-  - '**/sd-hwdb*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-hwdb*/**'
 sd-id128:
-  - '**/sd-id128*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-id128*/**'
 sd-netlink:
-  - '**/sd-netlink*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-netlink*/**'
 sd-path:
-  - '**/sd-path*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-path*/**'
 sd-resolve:
-  - '**/sd-resolve*/**'
+  - changed-files:
+    - any-glob-to-any-file: '**/sd-resolve*/**'
 selinux:
-  - '**/*selinux*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*selinux*'
 sleep:
-  - '**/*sleep*'
+  - changed-files:
+    - any-glob-to-any-file: '**/*sleep*'
```

Comment:
While at it, could you please change this to match files under `shared/` and `sleep/` only? It is painful to remove the spurious label triggered by e.g. `test/testsuite-03.units/sleep.service` currently.

Reply:
Will do, but I'll have to re-create this PR directly from a branch in the systemd repo, to be able to test this properly